### ISSUE TITLE
Add support for availability_type in read replica configuration

### ISSUE THREAD

**Issue by enocom** [terraform-google-modules/terraform-google-sql-db#328]
### TL;DR

The Terraform Google Cloud Provider supports specifying availability_type for read replicas, but the module does not (see [here](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/read_replica.tf#L23)).

This is a feature request to add the field for read replicas.

### Terraform Resources

```markdown
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#availability_type
```

### Detailed design

```markdown
I think this might just be as simple as adding `availability_type` to the https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/read_replica.tf#L38 resource.
```

**Comment by g-awmalik** [comment]
@enocom - is this request particularly for Postgres or for MySql as well?

**Comment by enocom** [comment]
Ideally all three engine types.

**Comment by g-awmalik** [comment]
The PR is merged but we'll have a wait a bit for v12.0.0 because there's an upstream provider issue that is causing us to delay the release. If the fix isn't in early next week, we'll cut a release anyway. For now, feel free to pin it to master to get this feature.

**Comment by enocom** [comment]
Thanks @g-awmalik!
